src/Ormolu/Printer/Meat/Declaration/Value.hs

```diff
@@ -549,8 +549,9 @@ p_hsExpr = \case
       breakpoint
       inci (p_hsExpr x)
   HsMultiIf NoExt guards -> do
-    txt "if "
-    sitcc $ sep newline (located' (p_grhs RightArrow)) guards
+    txt "if"
```

Comment:
Here, notice we first output the literal `"if"`, then we need either single space or newline, this is called `breakpoint`. Then we need to make sure that the rest is more indented, so we use `inci`. It has no effect in single-line case, and bump indentation as you'd expect in multiline case.

Reply:
Thanks! I will try to use them in future patches.